internal/command/output.go

```diff
@@ -75,6 +77,25 @@ func (c *OutputCommand) Outputs(statePath string) (map[string]*states.OutputValu
 		return nil, diags
 	}
 
+	// See if we are using the Cloud backend
+	cloud, cloudOk := b.(*cloud.Cloud)
+	if cloudOk {
+		workspaceOutputs, err := cloud.WorkspaceOutputs(env)
+		if err != nil {
+			diags = diags.Append(fmt.Errorf("Failed to load state: %s", err))
+		}
+		// Convert the []tfe.StateVersionOutput to map[string]OutputValue
+		rootModule := states.NewState().RootModule()
+		for _, output := range workspaceOutputs {
+			rootModule.SetOutputValue(
+				output.Name,
+				cty.StringVal(fmt.Sprintf("%v", output.Value)),
```

Comment:
I don't think this is correct. Output values can be of any type, not just string, so we need to decode the value from the Cloud API into an appropriate `cty.Value`. It might make sense to perform that decode in the `cloud` package rather than `command`, to consolidate the Cloud API related logic over there.

This is a similar issue to the assumption that variable values are always strings which caused problems when adding var file support to the Cloud integration.

Reply:
I've updated the PR with something that works with complex outputs. I'd love your feedback. I'll work on a cloud integration test in the meantime.